Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting.py

```diff
@@ -1,6 +1,7 @@
-import demistomock as demisto
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
 
-from CommonServerPython import *
```

Comment:
If its not pointed by any linter, lets drop this change, we rather have  minimum non-required changes.